contributors/design-proposals/api-machinery/admission-control-webhooks.md

```diff
@@ -254,6 +239,24 @@ consider mechanisms to make this easier. We expect to gather user feedback
 before designing this.
 
 
+## Conversion and Versioning
+
+Webhooks will recieve the admission review subject in the exact version which
+the user sent it to the control plane. This may require the webhook to
+understand multiple versions of those types.
+
+All communication to webhooks will be JSON formatted. We will likely change that
+for GA.
```

Comment:
json + admission.k8s.io/v1beta1 wrapper object for now
will be able to indicate desire for proto or admission.k8s.io/v1beta2, etc in the future